streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -71,65 +63,11 @@ public void testRemoveSensor() {
 
         final Sensor sensor3 = streamsMetrics.addThroughputSensor(scope, entity, operation, Sensor.RecordingLevel.DEBUG);
         streamsMetrics.removeSensor(sensor3);
-
-        assertEquals(Collections.emptyMap(), streamsMetrics.parentSensors());
-    }
-
-    @Test
-    public void testMutiLevelSensorRemoval() {
```

Comment:
We do not have by-default parent sensor now, and the rest of special parent sensors are covered in `StreamMetricsIntegrationTest` already.